e2e/cypress/integration/add_user_to_channel_from_profile_popover/profile_popover_spec.js

```diff
@@ -31,29 +37,12 @@ describe('Profile popover', () => {
         // # Do the system console setup
         cy.apiAdminLogin();
         cy.visit('/admin_console/user_management/permissions/system_scheme');
-        cy.findByTestId('all_users-public_channel-checkbox').scrollIntoView().should('be.visible').click();
```

Comment:
@saturninoabril - you were right. Clicking it did disable it. Removed it now. Not sure why it wasn't working yesterday.